lib/board.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class Board
+  attr_accessor :sq_val
```

Comment:
**[REQUIRED]**
Since you are modifying the sq_val only inside the Board class itself, you can use attr_reader instead. Encapsulation is important because it is a protective shield that prevents data from being accessed and / or altered by code outside that shield.